best-practices/README.md

```diff
@@ -13,6 +13,7 @@ General
 * Don't write code that guesses at future functionality.
 * Exceptions should be exceptional.
 * [Keep the code simple].
+* Good code is part of our culture.
```

Comment:
🤔

Reply:
Please refer to our Code Review Guidelines.

![Untitled](https://user-images.githubusercontent.com/5141/62502040-7a8a9c00-b7a2-11e9-89c4-a6f4ce286f95.png)

Reply:
NO SARCASM IS NO FUN :trollface:

Reply:
Do you want me change it?

Reply:
Don't think so, in all seriousness, it's probably a good rule to have. At least in PR comments.